src/target/move.c

```diff
@@ -250,33 +250,38 @@ void move_front(void)
 }
 
 /**
- * @brief Move left into the next cell.
+ * @brief Move left or right into the next cell.
  */
-void move_left(void)
+static void move_side(enum step_direction side)
```

Comment:
Create an issue to configure these variables.

Reply:
https://github.com/Theseus/bulebule/issues/286